src/ui/views/TeamHistory/index.tsx

```diff
@@ -59,7 +59,19 @@ const TeamHistory = ({
 					Schedule
 				</a>{" "}
 				|{" "}
-				<a href={helpers.leagueUrl(["news", `${abbrev}_${tid}`])}>News Feed</a>
+				<a href={helpers.leagueUrl(["news", `${abbrev}_${tid}`])}>News Feed</a>{" "}
+				|{" "}
+				<a href={helpers.leagueUrl(
+						season === undefined ? ["player_stats"] : [
+									"player_stats",
+									`${abbrev}_${tid}`,
+									"career",
+									process.env.SPORT === "basketball" ? "totals" : statType,
```

Comment:
`statType` does not exist. FBGM does the stat pages a little different than BBGM, it has no separate "totals" page but by default does show totals for most stats.

Change this to `process.env.SPORT === "football" ? "passing" : "totals"` and it will default to the passing stats for FBGM.